bayesian_cnn_prometheus/analysis/correlation_analyzer.py

```diff
@@ -0,0 +1,159 @@
+import json
+import numpy as np
+
+from bayesian_cnn_prometheus.evaluation.utils import load_nifti_file # refactor idea: move this function to more 'general' directory
+from skimage import metrics
+from matplotlib import pyplot as plt
+from scipy.signal import correlate
+
+
+class CorrelationAnalyzer:
+    def __init__(self, variance_mask_path: str, cancer_mask_path: str):
+        """
+        Creates CorrelationAnalyzer object to compare generated variance masks with cancer masks.
+        :param variance_mask_path: path to a file with generated variance masks
+        :param cancer_mask_path: path to a file with original cancer masks
+        """
+        self.variance_mask_path = variance_mask_path
+        self.cancer_mask_path = cancer_mask_path
```

Comment:
Może się czepiam, ale nie używałabym słowa "cancer" :D W sensie szukamy zmian chorobowych. Tutaj w przypadku płuc chyba faktycznie jest to nowotwór, ale na innych datasetach to będą różnego typu zmiany patologiczne. :)
Na pewno się czepiam :D